client/apps/eternum-mobile/src/pages/realm/ui/tabs/military-tab.tsx

```
@@ -1,16 +1,99 @@
+      bgColor: "bg-yellow-500/10",
+      icon: Swords,
+    };
+  return {
+    level: "low",
+    color: "text-green-500",
+    bgColor: "bg-green-500/10",
+    icon: Eye,
+  };
+};
+
+export function MilitaryTab({}: MilitaryTabProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)